### PR TITLE
Additional Machine Messages

### PR DESCRIPTION
**Pull request by TheConnMan**
Ref #79, #87

### CODE REVIEW

File: api/services/ChatService.js [hunk 1]
```diff
@@ -13,11 +13,7 @@ function getChats() {
 }
 
 function addUserMessage(chat) {
-  if (!chat.username) {
-    chat.username = 'Anonymous';
-  }
-  chat.type = 'user';
-  chat.time = new Date(chat.time).toISOString()
+  chat.time = new Date().toISOString();
```
Comment (SirZach) [inline api/services/ChatService.js]: this won't represent the time the user submitted it, it'll be the time when the server gets to processing it

Comment (TheConnMan) [inline api/services/ChatService.js]: Agreed, but that's more in sync with the machine messages generated by the server. What's the case where the client is better than the server for those times?

Comment (SirZach) [inline api/services/ChatService.js]: With actions like `skip` the real source of truth is when the server decides to skip the song. When it comes to chat, chat occurs at the moment the user hits enter

Comment (TheConnMan) [inline api/services/ChatService.js]: That's fair, I'll switch it back.

**Pull request merged**